### PR TITLE
[stable/grafana] Upgrade to Grafana 6.5.2

### PR DESCRIPTION
**Pull request by hairyhenderson**
## Is this a new chart

No

#### What this PR does / why we need it:

Upgrade to Grafana 6.5.2 - https://github.com/grafana/grafana/releases/tag/v6.5.2

#### Which issue this PR fixes

n/a

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

Signed-off-by: Dave Henderson <dhenderson@gmail.com>

### CODE REVIEW

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: grafana
-version: 4.1.3
-appVersion: 6.5.0
+version: 4.1.4
```
Comment (zanhsieh) [inline stable/grafana/Chart.yaml]: This version probably would fail since I just approved / lgtm your another PR. 4.2.1.

**Review by zanhsieh — COMMENTED** [review]
Please see if another your PR merged, and bump chart version respectively.

File: stable/grafana/Chart.yaml (hunk 1 above)
Comment (hairyhenderson) [reply stable/grafana/Chart.yaml]: done!

**Comment by hairyhenderson** [comment]
@zanhsieh version bumped - PTAL!

File: stable/grafana/values.yaml [hunk 2]
```diff
@@ -50,7 +50,7 @@ livenessProbe:
 
 image:
   repository: grafana/grafana
-  tag: 6.5.0
+  tag: 6.5.2
```
Comment (davidkarlsen) [inline stable/grafana/values.yaml:53]: please update readme to match too

Comment (hairyhenderson) [reply stable/grafana/values.yaml:53]: Done - noticed this hadn't been updated in a while 😅

**Comment by hairyhenderson** [comment]
Ok, version updated in `README.md` too (and rebased) - PTAL @davidkarlsen @zanhsieh 😁

**Pull request merged**